packages/cli/src/scripts/update.js

```diff
@@ -2,17 +2,17 @@ import _ from 'lodash';
 import stdout from '../utils/stdout';
 import ControllerFor from '../models/network/ControllerFor'
 
-export default async function update({ contractAlias, proxyAddress, initMethod, initArgs, all, network, force = false, txParams = {}, networkFile = undefined}) {
-  if (!contractAlias && !all) {
-    throw Error('The contract name to update must be provided, or explicit upgrading all proxies in the application.')
+export default async function update({ packageName, contractAlias, proxyAddress, initMethod, initArgs, all, network, force = false, txParams = {}, networkFile = undefined}) {
+  if (!packageName && !contractAlias && !proxyAddress && !all) {
+    throw Error('The package name, contract name, or address to update must be provided, or set the `all` flag to update all contracts in the application.')
   }
 
   const controller = new ControllerFor(network, txParams, networkFile)
 
   try {
     await controller.checkLocalContractsDeployed(!force);
-    const proxies = await controller.upgradeProxies(contractAlias, proxyAddress, initMethod, initArgs);
-    _(proxies).values().forEach(proxies => proxies.forEach(proxy => stdout(proxy.address)));
+    const proxies = await controller.upgradeProxies(packageName, contractAlias, proxyAddress, initMethod, initArgs);
+    _.forEach(proxies, proxy => stdout(proxy.address));
```

Comment:
Suggestion: You could use `proxies.forEach(proxy => stdout(proxy.address))` instead, and avoid lodash lib import (as this is the only lodash function call that is being used in this file). In addition to this, we could start importing only necessary lodash functions and not the whole lib (i.e., `import { forEach } from 'lodash'` or `import { forEach as _forEach } from 'lodash'` )